templates/nginx.conf

```diff
@@ -86,6 +101,20 @@ server {
       # Rate limit
       limit_req zone={{domain}}_ratelimit burst=30 nodelay;
 
+      #match /pictrs path
+      location ~ ^/(pictrs) {
+        proxy_cache lemmy_static_cache_{{domain}};
+        proxy_cache_use_stale error updating timeout http_500 http_502 http_503 http_504;
+
+        proxy_cache_revalidate on;
+        proxy_cache_background_update on;
+        proxy_cache_lock on;
+
+        proxy_cache_valid 200 1d; #only cache 200s
+
+        add_header X-Cache-Status $upstream_cache_status;
+      }
```

Comment:
Here its the same, pictrs should send the right cache headers. I thought it already does, maybe we need to use a newer version. Here is the project repo: https://git.asonix.dog/asonix/pict-rs/

Reply:
Also the images and assets are not causing much server load so Im not sure its worth the trouble caching them. My PR #75 caches API responses and HTML so it will reduce server load a lot as most users can be served from the cache instead of causing SQL queries.

Reply:
agreed. 
I did a test with pict-rs and it looks like it does some caching on it own as well. no need to have a cache on nginx side as well.

Reply:
Great, gonna close this pr then.